src/views/checkups/storage/components/hooks/useCheckupsStoragePermissions.ts

```diff
@@ -0,0 +1,85 @@
+import { useMemo } from 'react';
+import { findObjectByName } from 'src/views/checkups/utils/utils';
+
+import {
+  ClusterRoleBindingModel,
+  modelToGroupVersionKind,
+  RoleBindingModel,
+  RoleModel,
+  ServiceAccountModel,
+} from '@kubevirt-ui/kubevirt-api/console';
+import {
+  IoK8sApiCoreV1ServiceAccount,
+  IoK8sApiRbacV1ClusterRole,
+  IoK8sApiRbacV1ClusterRoleBinding,
+} from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { useActiveNamespace, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+import {
+  STORAGE_CHECKUP_ROLE,
+  STORAGE_CHECKUP_SA,
+  STORAGE_CLUSTER_ROLE_BINDING,
+} from '../../utils/utils';
+
+export const useCheckupsStoragePermissions = () => {
+  const [namespace] = useActiveNamespace();
+
+  const [serviceAccounts, loadingServiceAccounts] = useK8sWatchResource<
+    IoK8sApiCoreV1ServiceAccount[]
+  >({
+    groupVersionKind: modelToGroupVersionKind(ServiceAccountModel),
+    isList: true,
+    namespace,
```

Comment:
here namespace can be `ALL_NAMESPACE_SESSION_KEY`. Should we check that value as you did in other files?

Reply:
aqdded